IntrinsicGradedModules/examples/PresentationWithDegrees.g

```diff
@@ -17,7 +17,7 @@ Display( A );
 #! IntrinsicCategory( CategoryWithAmbientObjects(
 #! Freyd( GradedRows( Q[x,y,z] (with weights [ 1, 1, 1 ]) ) ) ) ):
 #! 
-#! 121 primitive operations were used to derive 312 operations for this category
```

Comment:
Have the installed methods for `ColiftOrFail` and `LiftOrFail` been buggy?

Reply:
I think this code was installed for `LiftOrFail` and `ColiftOrFail`:

https://github.com/homalg-project/CategoricalTowers/blob/1aacab2160331a4954ef249e70a8866397f96f7c/IntrinsicCategories/gap/IntrinsicCategories.gi#L1162-L1194

I have not checked if `Intrinsify` and `todo_func` can handle `fail`.